Hearthstone Deck Tracker/Controls/UnavailableBattlegroundsTypes.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<UserControl x:Class="Hearthstone_Deck_Tracker.Controls.UnavailableBattlegroundsTypes"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Hearthstone_Deck_Tracker.Controls" xmlns:hearthstone_deck_tracker="clr-namespace:Hearthstone_Deck_Tracker"
+             mc:Ignorable="d" 
+      d:DesignHeight="450" d:DesignWidth="800">
+
+    <Border BorderBrush="#141617" BorderThickness="1" Background="#23272a" Margin="0,5,0,0" Width="202" Visibility="{Binding UnavailableTypesVisibility, RelativeSource={RelativeSource AncestorType=local:UnavailableBattlegroundsTypes}}">
+        <StackPanel>
+            <Border Background="#1d3657" BorderBrush="#141617" BorderThickness="0,0,0,1">
+                <hearthstone_deck_tracker:HearthstoneTextBlock FontSize="14"
+                    Margin="7,2"
+                    Text="Missing Minion Types"
+                 />
+            </Border>
+            <Grid Grid.Row="0" ClipToBounds="True" Margin="5">
+                <TextBlock Text="{Binding UnavailableRacesText, RelativeSource={RelativeSource AncestorType=local:UnavailableBattlegroundsTypes}}" Foreground="White" Opacity="0.4" FontWeight="SemiBold" Visibility="{Binding UnavailableRacesVisibility, RelativeSource={RelativeSource AncestorType=local:UnavailableBattlegroundsTypes}}"/>
+            </Grid>
```

Comment:
Couple of changes / simplifications:
1. You don't need that wrapping grid. You can just set `Margin="5"` on the `TextBlock` directly.
2. The text is a bit too dark to be the prominent piece of information in this. `Opacity="0.6"` looks about good to me.
3. To align the text correctly with the header the left/right margin should be `7` rather than `5`. See `Margin="7,2"` above for the title. So `Margin="5"` => `Margin"7,5"`